ksql-version-metrics-client/src/test/java/io/confluent/ksql/version/metrics/collector/BasicCollectorTest.java

```diff
@@ -59,21 +61,25 @@ public void setCurrentTimeMillis(final long timeMillis) {
 
   private MockClock mockClock;
   private TimeUtils timeUtils;
+  private ActiveChecker activeChecker;
 
   @Before
   public void setUp() throws Exception {
     mockClock = new MockClock();
     timeUtils = new TimeUtils(mockClock);
+    activeChecker = new KsqlServerActiveCheckerImpl();
   }
 
   @Test
   public void testGetCollector() {
-    final BasicCollector basicCollector = new BasicCollector(moduleType, timeUtils);
+    activeChecker.onRequest(timeUtils.nowInUnixTime() - 15 * 60 * 1000, false);
```

Comment:
avoid coupling the unit tests of one class by relying on the implementation of another class, i.e. use a mock for the interface, rather than using instances of `KsqlServerActiveCheckerImpl`. 

The reason for this is that it keeps unit tests self-contained and you don't end up getting one unit test failing due to a bug in another class. (It's that classes unit test that should fail if there is a bug, not some other class's test).